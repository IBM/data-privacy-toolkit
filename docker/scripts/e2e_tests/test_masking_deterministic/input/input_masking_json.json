[
  {
    "id": "111-111-1111:1111-111-1111",
    "test": "111-111-1111:1111-111-1111",
    "testArraydoc": {
      "_id": "111-111-1111:1111-111-1111",
      "name": "name1"
    }
  },
  {
    "id": "111-111-1111:1111-111-1111",
    "test": "111-111-1111:1111-111-1111",
    "testArraydoc": {
      "_id": "111-111-1111:1111-111-1111",
      "name": "name2"
    }
  }
]