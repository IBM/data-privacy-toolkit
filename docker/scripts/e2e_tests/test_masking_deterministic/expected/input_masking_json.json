[{"id":"111-111-1111:1111-111-1111","test":"111-111-1111:1111-111-1111","testArraydoc":{"_id":"id-F2E2C9021FEBCFB461E5B6D3AA","name":"*****"}},{"id":"111-111-1111:1111-111-1111","test":"111-111-1111:1111-111-1111","testArraydoc":{"_id":"id-F2E2C9021FEBCFB461E5B6D3AA","name":"*****"}}]
